pkgs/development/interpreters/spidermonkey/68.nix

```diff
@@ -17,6 +17,13 @@ in stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
   setOutputFlags = false; # Configure script only understands --includedir
 
+  patches = optionals stdenv.isAarch32 [
```

Comment:
Applying patches conditionally is an anti pattern because they bit rot.

Reply:
While true, I feel like there's an argument to be made for avoiding dirtying derivations on platforms that do not need the patch, avoiding the usual staging churn, useless recompiles, etc. I'd be surprised if this isn't pulled in as part of some obscure hydra job anyway?